### ISSUE TITLE
Stuck at group picker - Structure dissect

### ISSUE THREAD

**Issue by Symbai** [cheat-engine/cheat-engine#933]
Adding over 200 addresses / columns to a structure dissect. Forms becomes laggy but that's okay. Right click at the first address and choose group option. All of that works with a small delay but that's okay. Group picker dialog opens but from now on, clicking on OK simply does nothing as well as clicking the X as well as hitting return or escape. I have to kill CE in taskmanager.

I think that the refresh of the values is totally screwing that up. Unless this is a bug I would like to suggest that either:

- Only the visible columns will be refreshed
- On dialog windows the refresh is paused until the dialog is closed

![ezgif com-optimize](https://user-images.githubusercontent.com/14368203/66425432-2f3d7580-e9ff-11e9-9a10-f3595a0fe697.gif)

**Comment by cheat-engine** [comment]
what is your update interval set to? Default is 500

**Comment by Symbai** [comment]
Update intervall 500, i7-4960X @ 3,6Ghz, Nvidia Geforce 1080, 32GB Ram

**Comment by Symbai** [comment]
It's not only group picker but ALL dialogs for example change update intervall. And even when the game is closed the dialogs cannot be closed. I hope this can be fixed.

**Comment by cheat-engine** [comment]
Is the debugger active during this?
If so, in settings tick the option to wait after updates

**Comment by Symbai** [comment]
Yes it is but why does it affect the structure dissect when it has lots of addresses?

**Comment by cheat-engine** [comment]
Multiple timers and threadevents(debugger) are executing getting put in front of 'less priority' event messages likes mouseclicks.
There may be events you don't see but do generate messages.
See if the delay for the debugger has some effect (besides slowing down the game/debugging)
